src/test/kotlin/org/austral/librawallet/account/TransferControllerTests.kt

```diff
@@ -0,0 +1,220 @@
+            Account(user = sender, balance = (initialSenderBalance * 100).toLong()),
+        )
+        val receiverAccount = accountRepository.save(
+            Account(user = receiver, balance = 0L),
+        )
+
+        val requestBody = """
+            {
+                "toIdentifier": "${receiverAccount.alias}",
+                "identifierType": "ALIAS",
+                "amount": $transferAmount
+            }
+        """.trimIndent()
+
+        mockMvc.perform(
+            post("/api/transfers")
+                .header("Authorization", senderToken)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(requestBody),
+        )
+            .andExpect(status().isConflict)
+            .andExpect(jsonPath("$.error").exists())
+    }
+
+    @Test
+    fun `AC3 transactions appear in both users histories via GET transactions`() {
+        val transferAmount = 15.50
+        val initialSenderBalance = 100.00
+        val initialReceiverBalance = 50.00
+
+        val (sender, senderToken) = createUserAndToken("sender3@example.com", "Pass5!")
+        val (receiver, receiverToken) = createUserAndToken("receiver3@example.com", "Pass6!")
+
+        val senderAccount = accountRepository.save(
+            Account(user = sender, balance = (initialSenderBalance * 100).toLong()),
+        )
+        val receiverAccount = accountRepository.save(
+            Account(user = receiver, balance = (initialReceiverBalance * 100).toLong()),
+        )
+
+        val requestBody = """
+            {
+                "toIdentifier": "${receiverAccount.alias}",
+                "identifierType": "ALIAS",
+                "amount": $transferAmount
+            }
+        """.trimIndent()
+
+        mockMvc.perform(
+            post("/api/transfers")
+                .header("Authorization", senderToken)
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(requestBody),
+        ).andExpect(status().isOk)
+
+        mockMvc.perform(
+            get("/api/transactions")
+                .header("Authorization", senderToken)
+                .contentType(MediaType.APPLICATION_JSON),
+        )
+            .andExpect(status().isOk)
+            .andExpect(jsonPath("$[0].type").value("EXPENSE"))
+            .andExpect(jsonPath("$[0].amount").value(transferAmount))
+
+        mockMvc.perform(
+            get("/api/transactions")
+                .header("Authorization", receiverToken)
+                .contentType(MediaType.APPLICATION_JSON),
+        )
+            .andExpect(status().isOk)
+            .andExpect(jsonPath("$[0].type").value("INCOME"))
+            .andExpect(jsonPath("$[0].amount").value(transferAmount))
+    }
+
+    @Test
+    fun `AC4-1 unauthenticated request yields 401`() {
+        val requestBody = """
+            {
+                "toIdentifier": "invalid",
+                "identifierType": "ALIAS",
+                "amount": 10.0
```

Comment:
Medio de gede, y si no querés no lo hagas, pero creo que sería más prolijo mover el `amount` a una variable.
```suggestion
        val amount = 10.0
        val requestBody = """
            {
                "toIdentifier": "invalid",
                "identifierType": "ALIAS",
                "amount": $amount
```